Formula/faust.rb

```diff
@@ -0,0 +1,29 @@
+class Faust < Formula
+  desc "Functional AUdio STream is language for signal processing and synthesis."
+  homepage "http://faust.grame.fr"
+  option "with-faust0", "install faust 0.x.x not faust 2.x.x"
```

Comment:
A formula should install a single version of the software. This option should go, and the stable 2 branch should be provided.